### PR TITLE
add zh_CN.po

### PR DESCRIPTION
**Pull request by yetist**
Add Chinese translation.

- [ ] de.po : (706 / 908 remain untranslated)
- [ ] es.po : (225 / 908 remain untranslated)
- [ ] fr.po : (689 / 908 remain untranslated)
- [ ] ru.po : (222 / 908 remain untranslated)
- [ ] zh_CN.po : (211 / 908 remain untranslated)

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -0,0 +1,5066 @@
...
+
+#: hardinfo/dmi_util.c:144
+msgid "Subnotebook"
+msgstr ""
+
+#: hardinfo/dmi_util.c:145
+#, fuzzy
+msgid "Space-saving"
+msgstr ""
+"强制保持相应的纹理设置为单个的纹理，而不是为了节省空间而由更小的独立纹理组成"
```
Comment (lpereira) [inline po/zh_CN.po]: Is this correct?

File: po/zh_CN.po [hunk 2]
```diff
@@ -0,0 +1,5066 @@
...
+
+#: hardinfo/dmi_util.c:130
+#, fuzzy
+msgid "Invalid chassis type (0)"
+msgstr "%s：无效的邮箱类型"
+
+#: hardinfo/dmi_util.c:131 hardinfo/dmi_util.c:132
+#, fuzzy
+msgid "Unknown chassis type"
+msgstr "unhook：未知钩子类型：%s"
```
Comment (lpereira) [inline po/zh_CN.po]: This doesn't look correct. Why is there a `%s`?

**Comment by lpereira** [comment]
Thanks for the translation! I don't read Chinese, so I can't really review the text; however, some of them do not seem right after passing them through an online automatic translation tool.

**Comment by mckaygerhard** [comment]
@lpereira for those laguajes that are 80% interpreted (pronunciation and intonation are important, which means that it depends on the context) a translation tool cannot detect the context or the intonation.. a light example are russian..

of course some cases has very good trasnlations like spanish in the google translate.. but not same for others.. so well, we must waith for someone that confirms or try to see what wil said @yetist about that

**Comment by bp0** [comment]
There are several strings marked fuzzy, and several that have c-format or access keys that shouldn't. I'd guess that a tool was used, but at least they marked the inexact strings as fuzzy, which means all the non-fuzzy strings are probably acceptable translations.

**Comment by mckaygerhard** [comment]
maybe due seems this file are from a ancient version of hardinfo..

**Comment by yetist** [comment]
Now drop the fuzzy item, and update some item, please review, thank you.
- [ ] zh_CN.po : (527 / 908 remain untranslated)

**Comment by mckaygerhard** [comment]
well @bp0 and @lpereira as i suspect was a translation of and older versino, now updated fuzzy lines gone but at least

**Comment by yetist** [comment]
This is not the old version, because I ran the updatepo.sh script, regenerated the pot file, and translated it based on the new pot file.
As for fuzzy, I use poedit to learn many translation entries, so there will be fuzzy entries.

File: po/zh_CN.po (hunk 1 above)
Comment (bp0) [reply po/zh_CN.po]: If this is true, it is hilarious.

**Comment by lpereira** [comment]
Merged, thanks!

**Pull request merged**